test/export/BUILD

```diff
@@ -0,0 +1,6 @@
+subinclude("//test/build_defs")
+
+plz_e2e_test(
+    name = "export_src_please_test",
+    cmd = "plz export --output plz-out/plzexport //src:please && plz --repo_root=$(plz query reporoot)/plz-out/plzexport build //src:please",
```

Comment:
might want to keep an eye on the performance of this guy. rebuilding the entire thing in a test could be expensive - unless it can utilise already cached artifacts?

Reply:
Yeah, the directory cache should mean these are all cached on the machine but I wonder if that's actually true. Might explain why the darwin build is timing out.

Reply:
Just reduced this to do //src/core only which has good coverage (a few subrepo and most of the same subincludes). From scratch the test takes about 20s on my machine. I think it's able to re-use stuff from the directory cache between the main build and the test.